sp800-63d/vot_mapping.md

```diff
@@ -40,26 +38,23 @@ IAL maps into the VoT component P using a numeric identifier. The implemented IA
 
 #### Identity Proofing Characteristics
 
-In addition, identity proofing characteristics MAY be indicated using an alphabetic identifier. Multiple alphabetic identifiers MAY be used simultaneously. Identity proofing methods or approaches that are required for the asserted IAL SHOULD NOT be asserted. The proofing method SHALL be sufficient or greater than that which is required by the IAL value indicated in the vector. 
-
-|Identity Proofing Method or Approach|Component Value|
+|Identity Proofing Characteristic|Component Value|
 |:----:|:--:|
 |In-person|Pi|
 |Remote|Pr|
 |Knowledge-based Verification (KBV)|Pk|
 |Address Confirmation with Postal Address|Pa|
 |Trusted Referee|Pt|
 |Additional features implemented beyond requirements for asserted IAL|Px|
+|**NOTE: We don't think Px conveys very much - because which values represent "approaches beyond those required under the asserted IAL"? Suggest removing this value. Does this comment also apply to Ax for federation?**||
```

Comment:
Worth keeping in draft form to see if we get feedback on it. I also don't think it adds much on its own but I can possibly see a desire for a general "I'm better than that" flag -- at least that was the original argument.